src/app/components/settings/background/background.component.ts

```diff
@@ -53,6 +55,10 @@ export class BackgroundComponent extends SubscriptionsComponent implements OnIni
 		});
 	}
 
+	EditLayout() {
```

Comment:
Typically, we use `camelCase` for variables, functions, etc. in JavaScript/TypeScript. We use `CapitalCamelCase` for classes or enums.